src/feedback/snackbar/snackbar.jsx

```diff
@@ -0,0 +1,100 @@
+import {
+    get,
+} from 'lodash';
+import ClassNames from 'classnames';
+import MUISnackbar from '@material-ui/core/Snackbar';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+    BEM_SNACKBAR,
+    UI_CLASS_NAME,
+} from '../../global/constants';
+import makeStyles from '../../styles/makeStyles';
+
+const propTypes = {
+    /**
+     * Override or extend the styles applied to the component.
+     * See [CSS API](https://material-ui.com/api/snackbar/#css) for more details.
+     */
+    classes: PropTypes.shape({}),
+    /**
+     * Assign additional class names to Mobile Stepper.
```

Comment:
Should say Snackbar instead of Mobile Stepper

Reply:
Fixed.

Reply:
@cbrewer1689 This one is not fixed.  I think this is worth fixing prior to merge.

Reply:
Weird, it was saved before the commit and push...

Reply:
There we go.  Looking good as of 264ca54.